src/length.rs

```diff
@@ -254,6 +254,11 @@ where
     }
 }
 
+impl<T, U> ValueOrLength<T, U> for Length<T, U> {
```

Comment:
please no! This is a hack, and essentially could be replaced by `Into<T>` but better not be replace at all.